inc/branding/namespace.php

```diff
@@ -233,7 +233,7 @@ function admin_bar_menu( WP_Admin_Bar $wp_admin_bar ) {
 	$logo_menu_args = [
 		'href'  => admin_url(),
 		'id'    => 'altis',
-		'title' => '<span class="altis-logo-wrapper"><img src="' . get_logo_url( 'white' ) . '" /></span>',
+		'title' => '<span class="altis-logo-wrapper"><img src="' . get_logo_url( 'white' ) . '" alt="Altis WordPress Admin" /></span>',
```

Comment:
Thanks for the suggestion, one minor change.
```suggestion
		'title' => '<span class="altis-logo-wrapper"><img src="' . get_logo_url( 'white' ) . '" alt="Altis Admin" /></span>',
```